src/main/java/com/ontology/service/impl/OntIdServiceImpl.java

```diff
@@ -160,7 +160,6 @@ public void run() {
                     Thread.sleep(6*1000);
                     Object event = sdk.checkEvent(txHash);
                     while (Helper.isEmptyOrNull(event)) {
-//                        sdk.addAttributes(ontId,password,key,valueType,value);
                         Thread.sleep(6*1000);
                         event = sdk.checkEvent(txHash);
```

Comment:
I think we need to set a upper limit for  executing `checkEvent`.

Reply:
https://github.com/ontio-ddxf/data-dealer/pull/7/commits/f922720a7aefb4bc27652a8405f35092b4a9fa5b